src/test/java/org/carlspring/cloud/storage/s3fs/S3FileSystemTest.java

```diff
@@ -448,8 +467,8 @@ public void isSameFile()
     private Map<String, ?> buildFakeEnv()
     {
         return ImmutableMap.<String, Object>builder().put(ACCESS_KEY, "access-key")
-                                                     .put(SECRET_KEY, "secret-key")
-                                                     .build();
+                                        .put(SECRET_KEY, "secret-key")
+                                        .build();
```

Comment:
This is a non-change, right? It seems to have changed the formatting.

Reply:
Yeah, sorry, I was making some tests. I'll revert it.